harvester/src/main/java/com/github/reflectoring/infiniboard/harvester/source/url/UrlSourceJob.java

```diff
@@ -56,11 +62,37 @@ protected void executeInternal(ApplicationContext context, JobKey jobKey, Map co
 
             upsertResults(context, jobKey, results);
 
+        } catch (SSLHandshakeException e) {
+            String msg = "Could not establish SSL connection to '%s'. '%s' is set to '%s'. Cause: '%s'";
+            logConnectionError(msg, url, enableSslVerify, e);
+        } catch (HttpHostConnectException e) {
+            String msg = "Could not establish connection to '%s'. '%s' is set to '%s'. Cause: '%s'";
+            logConnectionError(msg, url, enableSslVerify, e);
+        } catch (UnknownHostException e) {
```

Comment:
All we need to log is the exception message. I've checked it and this one is a false-positive.